css/maktum.css

```diff
@@ -0,0 +1,440 @@
+body {
+   
+    min-height: 100vh;
+    font: normal 16px sans-serif;
+  
+}
+
+  .carousel-inner img {
```

Comment:
Check whether this class is associated with bootstrap. If so, please redefine this class.
Note: Similar issues will be notified as IBC (Identical to bootstrap class)